src-tauri/tauri.conf.json

```diff
@@ -27,15 +27,16 @@
     "category": "DeveloperTool",
     "shortDescription": "",
     "longDescription": "",
-    "createUpdaterArtifacts": false
+    "createUpdaterArtifacts": "v1Compatible"
   },
   "plugins": {
     "updater": {
-      "active": false,
-      "dialog": false,
+      "windows": {
+        "installMode": "passive"
+      },
       "pubkey": "dW50cnVzdGVkIGNvbW1lbnQ6IG1pbmlzaWduIHB1YmxpYyBrZXk6IDM5Q0RCMDMyRkYwQTQxOTIKUldTU1FRci9NckROT1FERSs0UjhTNzRsTHU0cjZGdTZzN1FTWGF4aldYdGFlTG56eWc5Uko4YzkK",
       "endpoints": [
-        "https://raw.githubusercontent.com/open-goal/launcher/main/.tauri/latest-release-v2.json"
+        "https://github.com/open-goal/launcher/releases/latest/download/latest.json"
```

Comment:
revert any changes made to this file and apply them to https://github.com/open-goal/launcher/blob/0a4eb1901f3b5f292262ee2b01484982a635fd4d/src-tauri/tauri.conf.release.json instead